src/benchmarklib/abstract_table_generator.cpp

```diff
@@ -302,46 +304,17 @@ void AbstractTableGenerator::generate_and_store() {
   }
 
   /**
-   * Create indexes if requested by the user
+   * Create chunk indexes if requested by the user.
    */
-  if (_benchmark_config->indexes) {
-    std::cout << "- Creating indexes" << std::endl;
-    const auto& indexes_by_table = _indexes_by_table();
-    if (indexes_by_table.empty()) {
-      std::cout << "-  No indexes defined by benchmark" << std::endl;
-    }
-    for (const auto& [table_name, indexes] : indexes_by_table) {
-      const auto& table = table_info_by_name[table_name].table;
-
-      for (const auto& index_columns : indexes) {
-        auto column_ids = std::vector<ColumnID>{};
-
-        for (const auto& index_column : index_columns) {
-          column_ids.emplace_back(table->column_id_by_name(index_column));
-        }
-
-        std::cout << "-  Creating index on " << table_name << " [ ";
-        for (const auto& index_column : index_columns) {
-          std::cout << index_column << " ";
-        }
-        std::cout << "] " << std::flush;
-        Timer per_index_timer;
-
-        if (column_ids.size() == 1) {
-          table->create_index<GroupKeyIndex>(column_ids);
-        } else {
-          table->create_index<CompositeGroupKeyIndex>(column_ids);
-        }
-
-        std::cout << "(" << per_index_timer.lap_formatted() << ")" << std::endl;
-      }
-    }
-    metrics.index_duration = timer.lap();
-    std::cout << "- Creating indexes done (" << format_duration(metrics.index_duration) << ")" << std::endl;
+  if (_benchmark_config->chunk_indexes) {
```

Comment:
Here, we handle the conditional outside of `_create_chunk_indexes()`. In `_create_tables_indexes()` it is handled within the function. Why?

Reply:
Good point. The `_benchmark_config->table_indexes` was removed before, so we don't handle the conditional at all for `_create_table_indexes()`. I'll fix it.